scene/gui/line_edit.cpp

```diff
@@ -1211,6 +1211,9 @@ void LineEdit::delete_char() {
 }
 
 void LineEdit::delete_text(int p_from_column, int p_to_column) {
+	if (p_from_column < 0 || p_from_column > p_to_column || p_to_column > text.length()) {
+		return;
+	}
```

Comment:
Maybe this should throw an error instead, so that users know that they're using it wrong:
```
ERR_FAIL_COND_MSG(p_from_column < 0 || p_from_column > p_to_column || p_to_column > text.length(),
		vformat("Positional parameters (from: %d, to: %d) are inverted or outside the text length (%d).", p_from_column, p_to_column, text.length());
```